### ISSUE TITLE
Splash image is small

### ISSUE THREAD

**Issue by ewanyuan** [fayaz07/animated_splash#15]
Hi guys, could you help me? The splash image is small leaving other space white blank.
![image](https://user-images.githubusercontent.com/6346784/97834141-0c6cb600-1d12-11eb-995e-3e44f6399581.png)

code is:
home: AnimatedSplash(
        imagePath: 'static/img/splash.png',
        home: IndexPage(),
        customFunction: duringSplash,
        duration: 2500,
        type: AnimatedSplashType.BackgroundProcess,
        outputAndHome: returnValueAndHomeScreen,
      ),

**Comment by jdeltoft** [comment]
I agree, tried center and expand around the AnimatedSplash() but no luck so far.
